teloc/tests/example.rs

```diff
@@ -0,0 +1,46 @@
+use teloc::*;
```

Comment:
Added this test to ensure that the code in the example compiles

Reply:
You can test readme examples using such hack (taken from [teloxide](https://github.com/teloxide/teloxide/blob/master/src/lib.rs#L85-L87)):
```
#[cfg(all(feature = "nightly", doctest))]
#[cfg_attr(feature = "nightly", cfg_attr(feature = "nightly", doc = include_str!("../README.md")))]
enum ReadmeDocTests {}
```